apps/fz_http/lib/fz_http/devices/device.ex

```diff
@@ -2,20 +2,11 @@ defmodule FzHttp.Devices.Device do
   @moduledoc """
   Manages Device things
   """
-
   use Ecto.Schema
   import Ecto.Changeset
-  require Logger
-
-  import FzHttp.Validators.Common,
```

Comment:
I removed import because it adds compile-time dependency and we don't want to accumulate that over time.

Reply:
👍🏼